### PR TITLE
tooltips

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
and limit unnecessary renders on scenario viewer

### CODE REVIEW

File: gui/src/ScenarioViewer.tsx [hunk 1]
```diff
@@ -479,14 +476,9 @@ const ScenarioViewer = (props: ScenarioViewerProps) => {
                             </Grid>
                             <Grid item>
                                 {showSubmit ? (
-                                    <Tooltip title={disabled ? "Cannot submit Sequence" : "Submit Scenario"}>
+                                    <Tooltip title={disabled ? "Cannot submit Scenario" : "Submit Scenario"}>
```
Comment (FabienLelaquais) [inline gui/src/ScenarioViewer.tsx:471]: ? 
disabled is the discriminant between Scenarios and Sequences??

**Review by FabienLelaquais — DISMISSED** [review]
👍

File: gui/src/ScenarioViewer.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/src/ScenarioViewer.tsx:471]: fixed

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**